data/transactions/logic/assembler.go

```diff
@@ -969,7 +974,18 @@ func asmDefault(ops *OpStream, spec *OpSpec, args []string) error {
 	return nil
 }
 
-func typeSwap(pgm ProgramKnowledge, args []string) (StackTypes, StackTypes) {
+func getUintImm(args []string) (n uint64, errored bool) {
```

Comment:
You are probably right! I'm so used to seeing ParseUint called with 74 that I missed the 8.
In that case, let's cast it down to a byte when you're done to make the typing clearer.

Reply:
And if you do want to keep the boolean for the error, I'm ok with it, but I would reverse the meaning to "true=ok", because another common idiom is `parsed, ok := func()`